Puppetfile.stable

```diff
@@ -4,8 +4,8 @@ mod 'simp-doc',
   :git => 'https://github.com/simp/simp-doc',
   :ref => '79c18f732f6c73b8bff21809c639771a892722f2'
 
-mod 'simp-rsync',
-  :git => 'https://github.com/simp/simp-rsync',
+mod 'simp-rsync'
```

Comment:
The point of the ticket was to rename the module in the `mod` line. There are two modules named `simp-rsync`and r10k complains. This does not address that issue.